### ISSUE TITLE
Ensure users can control the visual presentation of text

### ISSUE THREAD

**Issue by gilleygoat** [web-infrastructure/doc_architecture#39]
The user should have comprehensive controls over the presentation of text within a webpage.

Ensure that:

-     Foreground and background colors can be selected by the user
-     The width of text is no more than 80 characters or glyphs (40 if [CJK](https://en.wikipedia.org/wiki/CJK_characters))
-     Text is not justified (aligned to both the left and the right margins)
-     Line spacing is at least a space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing
-     Text can be increased to up to 200% size without assistive technology, without requiring the user scrolls horizontally to read a line of text on a full-screen window

For further details, see [Understanding Success Criterion 1.4.8](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-visual-presentation.html).
